packages/mdc-select/foundation.ts

```diff
@@ -132,6 +132,14 @@ export class MDCSelectFoundation extends MDCFoundation<MDCSelectAdapter> {
     this.notchOutline(openNotch);
   }
 
+  handleMenuOpened() {
+    this.adapter_.addClass(cssClasses.ACTIVATED);
```

Comment:
I think the adapter methods should move to the menu, and should instead be adding `mdc-menu--selected`.

Reply:
This class name is added when select / menu dropdown is opened which is out of the scope of Menu component.